### PR TITLE
[W5.5b][W11-A2] Kevin Leonardo

### PR DESCRIPTION
**Pull request by k-l-a**
Pull Request for LO W5.5b Class Level Members.
Change parseCommand method and all other relevant methods in Parser to class level members(static).
Add nextSequenceNumber and sequenceNumber to Person.
Person now stores their sequence number based on order of creation.

### CODE REVIEW

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -9,6 +9,9 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
+    private static final int STARTING_SEQUENCE_NUM = 1;
```
Comment (JYL123) [inline src/seedu/addressbook/data/person/Person.java]: what if there are more than one person in the address book before this round of program starts?

Comment (k-l-a) [reply src/seedu/addressbook/data/person/Person.java]: When you load a storage file, every person will be recreated, so when you add a person the sequenceNum starts from *number of person in address book* + 1.
And every person in the address book will also have a sequenceNum (equal to their order in storage file).

Comment (k-l-a) [reply src/seedu/addressbook/data/person/Person.java]: That final variable is the number the 1st person created/added will have.
This includes the person(s) loaded from storage.
The 2nd, 3rd , and so on will just have `STARTING_SEQUENCE_NUM` + 1, 2, and so on.

Comment (JYL123) [reply src/seedu/addressbook/data/person/Person.java]: I got your point, thanks for explaining. Sorry, i think i commented on a wrong file. I am supposed to comment only on W5.4.

Comment (okkhoy) [reply src/seedu/addressbook/data/person/Person.java]: @JYL123 your last comment makes this PR appear in my filter causing me to think why is 5.5 here under 5.4 :joy:

**Comment by okkhoy** [comment]
@k-l-a 
Some comments added. Please ack & close the PR after reading comments.

**Comment by k-l-a** [comment]
Oh, you're right, I probably should have added a test for that.
Thank you for the review, PR will now be closed.

**Pull request closed without merging**